ttnn/cpp/ttnn/decorators.hpp

```diff
@@ -271,7 +271,7 @@ struct registered_operation_t {
                 const OptionalConstTensors& optional_input_tensors,
                 const OptionalTensors& optional_output_tensors) mutable {
                 auto execute_on_worker_thread_args = detail::map_launch_op_args_to_execute_on_worker_thread_args(
-                    input_tensors, optional_input_tensors, optional_output_tensors, std::forward<args_t>(args)...);
+                    input_tensors, optional_input_tensors, optional_output_tensors, args...);
```

Comment:
Would semantics be clearer to change the lambda capture to by-value instead of modifying it here?

Reply:
we already capturing it by value. I tried removing mutable modifier from lambda (compilation error).
Here is simplified example of what's going on: https://godbolt.org/z/rxvbP3o48
```
tt::tt_metal::operation::launch_op(
            [args...](
                const Tensors& input_tensors,
                const OptionalConstTensors& optional_input_tensors,
                const OptionalTensors& optional_output_tensors) mutable {
                auto execute_on_worker_thread_args = detail::map_launch_op_args_to_execute_on_worker_thread_args(
                    input_tensors, optional_input_tensors, optional_output_tensors, args...);
                return std::apply(
                    [](auto&&... args) {
                        return detail::map_execute_on_worker_thread_return_to_launch_op_return<operation_t>(
                            operation_t::invoke(std::forward<decltype(args)>(args)...));
                    },
                    execute_on_worker_thread_args);
            },
            input_tensors,
            output_tensors,
            optional_input_tensors,
            optional_output_tensors,
            enable_autoformat);
```